packages/mobile/src/harmony-native/components/input/TextInput/TextInput.tsx

```diff
@@ -349,10 +363,13 @@ export const TextInput = forwardRef(
                         css({
                           flex: 1,
                           // Need absolute height to ensure consistency across platforms
-                          height: !isSmall ? 23 : undefined,
+                          height: isExtraSmall ? 18 : !isSmall ? 23 : undefined,
```

Comment:
random nums

Reply:
gonna leave for now, we dont have 18 in harmony, but this number results in correct height according to designs